src/mmrelay/matrix/events.py

```
@@ -0,0 +1,738 @@
+
+        if original_matrix_event_id:
+            orig = await asyncio.to_thread(
+                facade.get_message_map_by_matrix_event_id, original_matrix_event_id
+            )
+            if not orig:
+                facade.logger.debug(
+                    "Original message for reaction not found in DB. Possibly a reaction-to-reaction scenario. Not forwarding."
+                )
+                return
+
+            meshtastic_id, matrix_room_id, meshtastic_text_db, meshtastic_meshnet_db = (
```

Ruff (0.15.9)

[warning] Unpacked variable `meshtastic_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)